examples/compatibility-6-wind-router-rules.yaml

```diff
@@ -0,0 +1,31 @@
+version: v1alpha1
```

Comment:
it's better to remove the company name in the file name, for example compatibility-nfd-rules.yaml